### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by RobertLebeda** [phplime/Qrexorder#3] [Ideas]
1. allow restaurants to create new subpages from the dashboard. for example, create a page for restaurant terms and conditions and a page for personal data protection.

2. enable the printing of the POS invoice via the bluetooth interface. Many POS order printing devices in kitchens are powered via bluetooth directly with an Android phone. or create a guide or another auxiliary application where the restaurant simply connects and accepts orders and prints using the phone.

3. allow restaurants to add a widget from elfsight.com cookie consent. it is very important for me, every party in Europe must have it. GDPR, otherwise I can't start the site and register users.

**Comment by phplime** [comment]
1.  terms and conditions have already added wait for the upcoming update.
2. Have a plan to add dynamic print 
3. Noted
Thank you!

**Reply by KewalT** [reply]
Great to hear this from you.

**Comment by RobertLebeda** [comment]
I would like to ask you to add a checkbox to your product update when registering or placing an order for new customers to whom the customer gives consent. 🙏🙏

text 1 must be ticked: I confirm that I have familiarized myself with the "Regulations" and the "Principles of personal data protection."
 
text 2 optional: (Voluntarily agree) I agree to the processing of my personal data for the purpose of marketing products and services of my own and cooperating entities.
"View information on personal data processing."

<img width="1237" alt="Snímka obrazovky 2023-05-03 o 20 33 53" src="https://user-images.githubusercontent.com/132360776/236012097-0b1e0b35-92d9-40b9-bd10-5cf70901a255.png">

**Reply by javaaid** [reply]
1. need more offline payments tabs
2. stock management should have a proper module.
3. reporting should have a proper module like other POS systems. (Full Reports)
4. Rider's live location

**Comment by RobertLebeda** [comment]
1. add link to footer ''created by ( link to admin homepage with logo) ''
2. add in the checkout the option to pay now, pay later and the option to pay by card on pickup ( optional option for restaurants) 
Reason: someone doesn't want an online payment gateway because they use their portable terminal from the bank for the delivery guy. Thank you
